### PR TITLE
[react-onsenui] RouterUtil and RouterNavigator, some more props

### PR DESCRIPTION
**Pull request by sbusch**
Add type definitions for RouterUtil and RouterNavigator, plus some more props

 ----

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] ~Add or edit tests to reflect the change.~ (Run with `npm test`.) No new tests (better to have at least some type defs which were missing). `npm test` run successfully.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://onsen.io/v2/api/react/> and source code of package
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed. -> was already there

### CODE REVIEW

File: types/react-onsenui/index.d.ts [hunk 1]
```diff
@@ -148,6 +150,38 @@ export class Navigator extends Component<{
     popPage(options?: PageTransitionOptions): Promise<HTMLElement>;
 }
 
+// Still incomplete, see https://onsen.io/v2/api/react/RouterNavigator.html
+export class RouterNavigator extends Component<{
+  routeConfig: any,
```
Comment (amcasey) [inline types/react-onsenui/index.d.ts:155]: I would naively have guessed that the type would be `RouteConfig` from below.

**Review by amcasey — CHANGES_REQUESTED** [review]
Can you please add some tests showing how these new declarations are consumed?

File: types/react-onsenui/index.d.ts (hunk 1 above)
Comment (sbusch) [reply types/react-onsenui/index.d.ts:155]: Most probably - and according to the docs - yes. See my general comment in the PR thread.

**Comment by sbusch** [comment]
I had no time adding tests, and now don't have access to the original project using OnsenUI anymore. This PR could be merged because it's still better than nothing, but this depends on the maintainers here.

I wouldn't mind if this PR would be rejected, though.

**Pull request merged**